client/python/gradio_client/utils.py

```diff
@@ -436,9 +451,14 @@ async def stream_sse_v0(
             headers=headers,
             cookies=cookies,
         ) as response:
-            async for line in response.aiter_text():
+            async for line in response.aiter_lines():
+                line = line.rstrip("\n")
+                if len(line) == 0:
+                    continue
                 if line.startswith("data:"):
                     resp = json.loads(line[5:])
+                    if resp["msg"] in [ServerMessage.log, ServerMessage.heartbeat]:
+                        continue
```

Comment:
Note that these changes were made in `stream_sse_v0`. Do you also want to make similar changes (particularly the heartbeat one) in `stream_sse_v1`?

Reply:
Whoops - yea I think it should be both!

Reply:
Actually the stream_sse_v1 case is handled in `Client.stream_messages`! Stream_sse_v1 doesn't directly connect to the SSE endpoint like v0 does. It just pulls messages off a message queue.